application-specfic/inzending-voor-toezicht/type-dossier-voor-inzending.ttl

```diff
@@ -320,6 +320,11 @@
   <http://www.w3.org/2004/02/skos/core#inScheme> <http://lblod.data.gift/concept-schemes/71e6455e-1204-46a6-abf4-87319f58eaa5>;
   <http://www.w3.org/2004/02/skos/core#topConceptOf> <https://data.vlaanderen.be/id/conceptscheme/BesluitDocumentType> .
 
+# "Vaststelling gemeentelijk beleidskader (gemeentewegen)"
+<https://data.vlaanderen.be/id/concept/BesluitType/b0fcc0c3-bb33-427f-8da2-4ef3833c9060>
+  <http://www.w3.org/2004/02/skos/core#inScheme> <http://lblod.data.gift/concept-schemes/71e6455e-1204-46a6-abf4-87319f58eaa5> ;
+  <http://www.w3.org/2004/02/skos/core#topConceptOf> <http://lblod.data.gift/concept-schemes/71e6455e-1204-46a6-abf4-87319f58eaa5> .
```

Comment:
It also seems different from how it's defined here: https://github.com/Informatievlaanderen/OSLOthema-lokaleBesluiten/blob/master/codelijsten/besluit-type.ttl#L522-L527

Reply:
@Windvis I remember you having a conversation about this, in de sprint discussion. Did it get resolved?

Reply:
I think that was about the use of `BesluitDocumentType` or `BesluitType`. I forgot to ask Nele so that still needs to be verified, indeed. My bad.

This question is about the difference between the codelist repo and the informatievlaanderen repo. It seems they add different triples? The setup seems different?

Reply:
Yeah I'm not an expert in all the data-types. Think this is more of a @cecemel question. But it could be due to the Loket being so "old". Started using this concept, before it was generalized.

Reply:
@Windvis @HugaertsDries some forms are not really besluiten (e.g. besluitenlijst, agenda, notulen), they are added in https://data.vlaanderen.be/id/conceptscheme/BesluitDocumentType

Does it help?